docs/Makefile

```diff
@@ -77,17 +77,17 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/gclouddatastore.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/google-cloud.qhcp"
 	@echo "To view the help file:"
-	@echo "# assistant -collectionFile $(BUILDDIR)/qthelp/gclouddatastore.qhc"
+	@echo "# assistant -collectionFile $(BUILDDIR)/qthelp/google-cloud.qhc"
 
 devhelp:
 	$(SPHINXBUILD) -b devhelp $(ALLSPHINXOPTS) $(BUILDDIR)/devhelp
 	@echo
 	@echo "Build finished."
 	@echo "To view the help file:"
-	@echo "# mkdir -p $$HOME/.local/share/devhelp/gclouddatastore"
-	@echo "# ln -s $(BUILDDIR)/devhelp $$HOME/.local/share/devhelp/gclouddatastore"
+	@echo "# mkdir -p $$HOME/.local/share/devhelp/google-cloud"
+	@echo "# ln -s $(BUILDDIR)/devhelp $$HOME/.local/share/devhelp/google-cloud"
```

Comment:
We never use the `Makefile`. That content was in there when the original name of the project was `gcloud-datastore`.